src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="GET" 
+    data-qa="big" 
+    class="form"
+    >
+      <input 
+      class="searchInput bigSearchInput" 
+      type="search" 
+      placeholder="Try &quot;Los Angeles&quot;" 
+      data-qa="keypress"
+      >
+      <img 
+      src="images/Search.svg" 
+      alt="searchIcon" 
+      class="bigSearchIcon icon"
+      >
+    </form>
```

Comment:
Do u wanna create hell for another dev?)

point 5 https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md